readme.md

```diff
@@ -169,6 +169,22 @@ The backend of this project is built using [FastAPI](https://fastapi.tiangolo.co
 | `JWT_SECRET` | _CHANGEME_ | JWT-based auth secret key, used in the process of signing a token |
 | `MAGIC_LINK_CHAT_TEMPLATE` | _CHANGEME_ | A template for magic link message, e.g. "You can login into APP_NAME by clicking this link: {magic_link}" |
 
+
+#### Twilio
```

Comment:
Please search the document for `TWILIO_BOT_PORT` and also update the bit about firewall rules. We no longer need those.